packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -36,7 +36,11 @@ The user can edit appointments as follows:
 
 ## Basic Setup
 
-Add the plugins listed above to the Scheduler and handle the `EditingState` plugin's `onCommitChanges` event to commit edits to the data storage.
+To enable editing, follow the steps below:
+
+1. Add the `EditingState` plugin and handle its `onCommitChanges` event to commit edits to the data storage.
+2. Add either `IntegratedEditing` or `EditRecurrenceMenu` plugin.
```

Comment:
How do I choose which plugin should I add?

Reply:
The `EditRecurrenceMenu` provides the capability to edit recurrent appointments, the `IntegratedEditing` does not. Should I clarify this? For example, "Add the `EditRecurrenceMenu` plugin if your data set may contain recurrent appoinments. Otherwise, add еру `IntegratedEditing` plugin"

Reply:
```suggestion
2. Add the `EditRecurrenceMenu` plugin if recurring appointments are allowed. Otherwise, add the `IntegratedEditing` plugin.
```